integration_tests/test_basic.py

```diff
@@ -723,3 +727,57 @@ def test_replay_protection(cronos):
         match=r"only replay-protected \(EIP-155\) transactions allowed over RPC",
     ):
         w3.eth.send_raw_transaction(HexBytes(raw))
+
+
+def test_submit_any_proposal(cronos, tmp_path):
+    # governance module account as granter
+    cli = cronos.cosmos_cli()
+    granter_addr = "crc10d07y265gmmuvt4z0w9aw880jnsr700jdufnyd"
+    grantee_addr = cli.address("signer1")
+
+    # this json can be obtained with `--generate-only` flag for respective cli calls
+    proposal_json = {
+        "messages": [
+            {
+                "@type": "/cosmos.feegrant.v1beta1.MsgGrantAllowance",
+                "granter": granter_addr,
+                "grantee": grantee_addr,
+                "allowance": {
+                    "@type": "/cosmos.feegrant.v1beta1.BasicAllowance",
+                    "spend_limit": [],
+                    "expiration": None,
+                },
+            }
+        ],
+        "deposit": "1basetcro",
+    }
+    proposal_file = tmp_path / "proposal.json"
+    proposal_file.write_text(json.dumps(proposal_json))
+    rsp = cli.submit_gov_proposal(proposal_file, from_="community")
+    assert rsp["code"] == 0, rsp["raw_log"]
+
+    # get proposal_id
+    ev = parse_events(rsp["logs"])["submit_proposal"]
+    print(ev)
+    proposal_id = ev["proposal_id"]
+    print("gov proposal submitted", proposal_id)
+
+    # not sure why, but sometimes can't find the proposal immediatelly
```

Comment:
yeah, that's normal, because tx broadcast only wait for included in mempool.

Reply:
This comment is removed in the latest commit.